alembic/env.py

```diff
@@ -19,24 +19,20 @@
 # for 'autogenerate' support
 # from myapp import mymodel
 # target_metadata = mymodel.Base.metadata
-<<<<<<< HEAD
-import os
-import sys
-sys.path.append(os.getcwd())
-=======
 # target_metadata = None
 
 sys.path.append(os.getcwd())
 config.set_main_option('sqlalchemy.url', os.getenv('DEV_DATABASE_URL'))
 
->>>>>>> 886bd3f4174947f2fb32aba10b8f635d69cbd92a
```

Comment:
This seems to be inappropriate code, please this was fixed, have you rebased this branch with latest develop?